src/App.tsx

```diff
@@ -8,32 +10,46 @@ function getRandomName(): string {
 }
 
 export const App: React.FC = () => {
-  const today = new Date();
-  let clockName = 'Clock-0';
-
-  // This code starts a timer
-  const timerId = window.setInterval(() => {
-    clockName = getRandomName();
-  }, 3300);
-
-  // this code stops the timer
-  window.clearInterval(timerId);
+  const [clockName, setTimerName] = useState('Clock-0');
+  const [hasClock, setClockOn] = useState(true);
+  const [prevClockName, setPrevName] = useState('Clock-0');
+
+  let timerId = 0;
+
+  const detectKeyDown = (e: MouseEvent) => {
```

Comment:
```suggestion
  const handleKeyDown = (e: MouseEvent) => {
```
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364